src/main/drivers/accgyro_mpu.c

```diff
@@ -224,60 +227,77 @@ bool mpuCheckDataReady(gyroDev_t* gyro)
 }
 
 #ifdef USE_SPI
-static bool detectSPISensorsAndUpdateDetectionResult(gyroDev_t *gyro)
+static bool detectSPISensorsAndUpdateDetectionResult(gyroDev_t *gyro, const sensorBus_t *bus)
 {
 #ifdef USE_GYRO_SPI_MPU6000
-    if (mpu6000SpiDetect()) {
+#ifdef MPU6000_CS_PIN
+    gyro->bus.spi.csnPin = bus->spi.csnPin == IO_NONE ? IOGetByTag(IO_TAG(MPU6000_CS_PIN)) : bus->spi.csnPin;
```

Comment:
IO_NONE is not necessary. If is designed so that you can simply zero out the memory to get 'no pin' value. 
Using it as 'use default' is fine; it will eventually go away anyway(there is little use for gyro-specific CS)